boards: arm: nrf: Convert to DT_LABEL() macro

galak opened the pull request:
Convert old dts_fixup.h style DT_GPIO_P0_DEV_NAME macro to new
DT_LABEL(DT_NODELABEL(gpio0))

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

galak committed:
boards: arm: nrf: Convert to DT_LABEL() macro

Convert old dts_fixup.h style DT_GPIO_P0_DEV_NAME macro to new
DT_LABEL(DT_NODELABEL(gpio0))

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

ioannisg commented:
@galak you could try to remove the warning here; those with the >80 characters seem easy; the others, I don't know :)

galak committed:
soc: arm: nordic_nrf: remove DT_GPIO_P._DEV_NAME define

All users of DT_GPIO_P._DEV_NAME have been converted so we can remove
these defines from dts_fixpu.h

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

galak commented:
ci failure is in qemu mps2 run, not related.  Going to merge and not re-run ci.
